### PR TITLE
chore: document rate-limits

### PR DESCRIPTION
**Pull request by tricky42**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Comment by tkjwa** [comment]
Do the rate limit on `whoami` also apply when using [Edge sessions](https://www.ory.sh/docs/concepts/cache) ?

File: docs/guides/rate-limits.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+id: rate-limits
+title: Project Rate Limits
+sidebar_label: Rate Limits
+---
+
+Ory Network currently enforces three rate limit policies based on the path of your request.
+
+1. `/sessions/whoami`: a dedicated rate limit policy for session checks.
+2. `/identities`: a dedicated rate limit policiy for identity inserts and updates.
```
Comment (vinckr) [inline docs/guides/rate-limits.mdx:10]:
```suggestion
2. `/identities`: a dedicated rate limit policy for identity inserts and updates.
```

**Review by vinckr — COMMENTED** [review]
looks good. 
should we also link it here https://www.ory.sh/docs/ecosystem/security and on h1 for our hackers?

also should we add a disclaimer that these rates are subject to change, the documentation might not always reflect the latest state and is more to give you a ballpark idea?

**Comment by tricky42** [comment]
> Do the rate limit on `whoami` also apply when using [Edge sessions](https://www.ory.sh/docs/concepts/cache)?
Yes, the rate limit on `whoami` applies when using edge sessions, as we trigger a revalidation with each request. In the future, we will further improve the logic here. The limits for `whoami`, therefore, are significantly higher than the other limits.

File: docs/guides/rate-limits.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+id: rate-limits
+title: Project Rate Limits
+sidebar_label: Rate Limits
+---
+
+Ory Network currently enforces three rate limit policies based on the path of your request.
+
+1. `/sessions/whoami`: a dedicated rate limit policy for session checks.
+2. `/identities`: a dedicated rate limit policy for identity inserts and updates.
```
Comment (zepatrik) [inline docs/guides/rate-limits.mdx:10]: We have GET and POST on this path, the details are a bit more complex.
IMO fine for a first version though.

**Pull request merged**